crates/yttrium/src/transaction/send/safe_test.rs

```diff
@@ -449,11 +450,40 @@ mod tests {
         Ok(user_operation_hash)
     }
 
-    #[tokio::test]
-    async fn test_send_transaction() -> eyre::Result<()> {
-        let rpc_url = Config::local().endpoints.rpc.base_url;
-        let rpc_url: reqwest::Url = rpc_url.parse()?;
-        let provider = ReqwestProvider::<Ethereum>::new_http(rpc_url);
+    async fn use_faucet(
+        provider: ReqwestProvider,
+        faucet: LocalSigner<SigningKey>,
+        amount: U256,
+        to: Address,
+    ) -> eyre::Result<()> {
+        // Basic check (which we can tune) to make sure we don't use excessive
+        // amounts (e.g. 0.1) of test ETH. It is not infinite, so we should use
+        // the minimum amount necessary.
+        assert!(amount < U256::from(20), "You probably don't need that much");
+
+        ProviderBuilder::new()
+            .with_recommended_fillers()
+            .wallet(EthereumWallet::new(faucet))
+            .on_provider(provider.clone())
+            .send_transaction(
+                TransactionRequest::default().with_to(to).with_value(amount),
+            )
+            .await?
+            .watch()
+            .await?;
+        let balance = provider.get_balance(to).await?;
+        assert_eq!(balance, amount);
+
+        Ok(())
+    }
```

Comment:
nice! we should share this with all tests I think 💯

Reply:
Yeah welcome to use it!